### DISCUSSION TITLE
Server Manager won't open

### DISCUSSION THREAD

**Discussion by EthanKC** [nextgenhealthcare/connect#5051] [Q&A] [answered]
I've recently installed Mirth on a VM. I'm able to start the server, and I've already built and tested some stuff using the Administrator. But I can't seem to open the Server Manager. Whenever I try, it gives me the UAC popup, where I click "yes". Then it does create a "mcmanager" background process, but nothing else happens. 

I'm running version  3.12.0 on Windows Server 2019 Datacenter

**Comment by pacmano1** [comment, marked as answer]
Is it not in the task tray on the lower right?

**Reply by EthanKC** [reply]
Why yes, yes it is, and I know realize that I had read previously that's where it would show up and forgot. Thanks!
